daemon/main.c

```diff
@@ -36,7 +36,7 @@ void netdata_cleanup_and_exit(int ret) {
         // exit cleanly
 
         // stop everything
-        info("EXIT: stopping master threads...");
+        info("EXIT: stopping main threads...");
```

Comment:
Typically there's one 'main' thread and this log message can be misinterpreted/confusing.